src/content/docs/distribute/debian.mdx

```diff
@@ -3,12 +3,44 @@ title: Debian
 sidebar:
   order: 1
   badge:
-    text: Stub
+    text: WIP
     variant: caution
 ---
 
 {/* Is this about building .deb files, creating PPAs, or adding something to the debian/ubuntu repos? */}
 
-import Stub from '@components/Stub.astro';
+## Debian
 
-<Stub />
+The stock Debian package generated by the Tauri bundler has everything you need to ship your application to Debian-based Linux distributions, defining your application's icons, generating a Desktop file, and specifying the dependencies `libwebkit2gtk-4.0-37` and `libgtk-3-0`, along with `libappindicator3-1` if your app uses the system tray.
+
+### Custom Files
+
+Tauri exposes a few configurations for the Debian package in case you need more control.
+
+If your app depends on additional system dependencies you can specify them in `tauri.conf.json > bundle > linux > deb`.
+
+To include custom files in the Debian package, you can provide a list of files or folders in `tauri.conf.json > bundle > linux > deb > files`. The configuration object maps the path in the Debian package to the path to the file on your filesystem, relative to the `tauri.conf.json` file. Here's an example configuration:
+
+```json
+{
+  // ... other configurations 
+  "bundle": {
+    "active": true,
+    "targets": "all",
+    "icon": [
+      "icons/32x32.png",
+      "icons/128x128.png",
+      "icons/128x128@2x.png",
+      "icons/icon.icns",
+      "icons/icon.ico"
+    ],
+    "linux": {
+      //... all linux related configurations go here 
```

Comment:
I think the v1 example was simpler to understand with example files (https://v1.tauri.app/v1/guides/building/linux#custom-files):
```
  "files": {
          "/usr/share/README.md": "../README.md", // copies the README.md file to /usr/share/README.md
          "usr/share/assets": "../assets/" // copies the entire assets directory to /usr/share/assets
        }
```